runtime/parachains/src/session_info/migration.rs

```diff
@@ -19,4 +19,74 @@
 use frame_support::traits::StorageVersion;
 
 /// The current storage version.
-pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
+pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(2);
+
+pub mod v2 {
+	use super::STORAGE_VERSION;
+	use crate::{
+		session_info,
+		session_info::{Pallet, Store},
+	};
+	use frame_support::{
+		pallet_prelude::Weight,
+		traits::{OnRuntimeUpgrade, StorageVersion},
+	};
+	use frame_system::Config;
+	use sp_core::Get;
+
+	#[cfg(feature = "try-runtime")]
+	use primitives::vstaging::ExecutorParams;
+
+	#[cfg(feature = "try-runtime")]
+	use crate::session_info::Vec;
+
+	#[cfg(feature = "try-runtime")]
+	use crate::shared;
+
+	/// The log target.
+	const TARGET: &'static str = "runtime::session_info::migration::v2";
```

Comment:
Why not name it `LOG_TARGET`. Is more clear and would also conform to what we use everywhere else.

Reply:
A copy-paste from an older migration of another pallet... I'll fix it.